.ci/scripts/check_c10_sync.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -exu
+ls pytorch/.git || git clone https://github.com/pytorch/pytorch.git
```

Comment:
nit: `--depth 1`? or will the commit checkout not work?

Reply:
pretty sure --depth 1 will break checking out specific commit. also checked and there's no way to clone just a specific commit